lib/core/viewer/components/pdf-viewer.component.ts

```diff
@@ -329,6 +331,22 @@ export class PdfViewerComponent implements OnChanges, OnDestroy {
         return document.getElementById(`${this.randomPdfId}-viewer-viewerPdf`);
     }
 
+    checkPageFitInContainer(scale: number) {
```

Comment:
can you rewrite this method to have a single exit and add the return type?

Reply:
Yes I will rewrite it